src/coord/src/coord.rs

```diff
@@ -1398,6 +1433,26 @@ where
                     }
                 }
             }
+        } else if let Some(_) = self.sources.get_mut(name) {
```

Comment:
Style nit, but this could be `self.sources.get(name).is_some()` I think, rather than the let binding that doesn't bind.

Reply:
:+1: